hivemind/proto/p2pd.proto

```diff
@@ -147,8 +159,9 @@ message PSRequest {
   optional bytes data = 3;
 }
 
+
 message PSMessage {
-  optional bytes from_id = 1;
+  optional bytes from = 1;
```

Comment:
Looks like this name is not used - maybe keep it as is to minimize diff?

```suggestion
  optional bytes from_id = 1;
```

Reply:
Let's leave it consistent with the [current version](https://github.com/libp2p/go-libp2p-daemon/blob/bfa03bdcc0e24a9663575648cd7b52a01fe3f42f/pb/p2pd.proto#L147) of p2pd.